tests/quantization/quark_integration/test_quark.py

```diff
@@ -83,7 +85,7 @@ def tearDown(self):
         avoid unexpected behaviors. Please see: https://discuss.pytorch.org/t/how-can-we-release-gpu-memory-cache/14530/27
         """
         gc.collect()
-        torch.cuda.empty_cache()
+        backend_empty_cache(torch_device)
```

Comment:
same comment if you want to make the change

Reply:
done